### PR TITLE
Decimals/gauge

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes #

_Briefly describe your proposed changes:_
_What was the problem?_
_What was the solution?_

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] swagger.json updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/shared/components/Gauge.tsx [hunk 1]
```diff
@@ -66,8 +66,8 @@ class Gauge extends Component<Props> {
     this.resetCanvas(canvas, ctx)
 
     const centerX = canvas.width / 2
-    const centerY = (canvas.height / 2) * 1.13
-    const radius = (Math.min(canvas.width, canvas.height) / 2) * 0.5
+    const centerY = canvas.height / 2 * 1.13
```
Comment (ischolten) [inline ui/src/shared/components/Gauge.tsx]: yay! nice

File: ui/src/dashboards/components/GraphOptionsDecimalPlaces.tsx [hunk 2]
```diff
@@ -42,6 +42,7 @@ class GraphOptionsDecimalPlaces extends PureComponent<Props> {
         <label> Decimal Places </label>
         <OptIn
           min="0"
+          max="20"
```
Comment (ischolten) [inline ui/src/dashboards/components/GraphOptionsDecimalPlaces.tsx]: Would you consider making this a constant in the constants file?

File: ui/src/shared/components/ClickOutsideInput.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 
 interface Props {
   min?: string
+  max?: string
```
Comment (ischolten) [inline ui/src/shared/components/ClickOutsideInput.tsx]: does this need a default value?

Comment (ebb-tide) [reply ui/src/shared/components/ClickOutsideInput.tsx]: no it doesn't have a max if it is not set.

File: ui/src/dashboards/components/GraphOptionsDecimalPlaces.tsx (hunk 2 above)
Comment (ebb-tide) [reply ui/src/dashboards/components/GraphOptionsDecimalPlaces.tsx]: will do thanks!

File: ui/src/shared/components/Gauge.tsx [hunk 4]
```diff
@@ -306,14 +308,34 @@ class Gauge extends Component<Props> {
...
+
+    if (decimalPlaces.isEnforced) {
+      valueString = value.toLocaleString(undefined, {
+        minimumFractionDigits: 0,
+        maximumFractionDigits: decimalPlaces.digits,
+      })
+    } else {
+      valueString = value.toLocaleString(undefined, {
+        minimumFractionDigits: 0,
+        maximumFractionDigits: 20,
```
Comment (ischolten) [inline ui/src/shared/components/Gauge.tsx]: Could this use the constant  mentioned above?

Comment (ebb-tide) [reply ui/src/shared/components/Gauge.tsx]: one is a string and one is a number but sure.

**Pull request merged**